### ISSUE TITLE
linters do not include package reference in errors

### ISSUE THREAD

**Issue by tfoote** [ament/ament_lint#5]
A test result like this is not particularly useful to browse the results and find the source code at issue. Often the name can be guessed by the duplicated package name in the path not always. 

Example from copyright: http://54.183.26.131:8080/job/ros2_batch_ci/83/testReport/(root)/missing%20copyright/__test_test_type_py/
```
missing copyright../test/test_type.py (from copyright)

Failing for the past 3 builds (Since Failed#81 )
Took 0 ms.
add description
Stacktrace

could not find copyright reference
```

Example from clang_format: http://54.183.26.131:8080/job/ros2_batch_ci/83/testReport/(root)/replacement/__src_functions_cpp_15624/

```
replacement../src/functions.cpp:15624 (from clang_format)

Failing for the past 1 build (Since Unstable#83 )
Took 0 ms.
add description
Stacktrace

Replace 1 characters with []
```

**Comment by dirk-thomas** [comment]
This should be addressed for our linters with #7, see http://54.183.26.131:8080/job/ros2_batch_ci/95/testReport/connext_cmake_module/copyright/__cmake_Modules_FindConnext_cmake/

But external tools like nosetests won't provide any kind of grouping, see http://54.183.26.131:8080/job/ros2_batch_ci/95/testReport/(root)/test_validate_field_types/test_validate_field_types/

**Comment by wjwwood** [comment]
This was partially done, because our linters are fixed, but nosetest cannot be changed without post processing, which we will not do for now.
